frontend/src/date-time/changeModifiedDate.ts

```diff
@@ -0,0 +1,10 @@
+import { DocumentDto } from '@shared/dto';
+import dateAndTime from './dateAndTime';
+import { documentEditVuexModule } from '@src/vuex/store-accessor';
+
+const changeModifiedDate = (document: DocumentDto): void => {
+  const date = dateAndTime();
+  documentEditVuexModule.changeModifiedDate([document, date]);
```

Comment:
You want to move some function to an external file only if this function is [pure](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976). So this function can be reused and tested. Function changeModifiedDate is not pure. If this code is duplicated in two+ components then move it to the vuex @Action. If this code is used in only one component then save it there.

Reply:
Done